opentracing_utils/libs/requests_.py

```diff
@@ -35,14 +40,21 @@ def requests_send_wrapper(self, request, **kwargs):
     if request_span:
         (request_span
             .set_operation_name(op_name)
-            .set_tag(opentracing_tags.HTTP_URL, request.url)
+            .set_tag(opentracing_tags.HTTP_URL, sanitize_url(request.url))
             .set_tag(opentracing_tags.HTTP_METHOD, request.method))
 
+        logger.debug('Opentracing requests: request_span={} parent={} trace={}'.format(
+            request_span.context.span_id, request_span.parent_id, request_span.context.trace_id))
+
         # Inject our current span context to outbound request
         try:
             carrier = {}
             opentracing.tracer.inject(request_span.context, Format.HTTP_HEADERS, carrier)
             request.headers.update(carrier)
+
+            # TODO: Leaky header logs - remove after done testing!
```

Comment:
do you need to finally remove this TODO? Is testing done?

Reply:
Was planning to test in deployment, but now I realize it was a Friday evening decision. I will just propagate the carrier values in the tags.
Will remove the log lines.